views/helpers/index.js

```diff
@@ -44,13 +44,60 @@ exports.moment = function(date) {
   return moment(date).fromNow();
 };
 
+/*
+  Light-weight helper function to generate
+  highlighted JSON strings
+*/
+function syntaxHighlight(json) {
```

Comment:
We need to think about whether we should actually do this at all (see https://github.com/GoogleChrome/gulliver/issues/180).